core/message_handler.go

```diff
@@ -43,13 +43,13 @@ var _ p2p.MessageHandler = &App{}
 func (app *App) GetMessagesToShare(max int) ([][]byte, error) {
 	// For now, we just select a random set of orders from those we have stored.
 	// TODO(albrow): This could be made more efficient if the db package supported
-	// a `Count` method for counting the number of models in a collection or
-	// counting the number of models that satisfy some query.
-	// TODO(albrow): Add an index for IsDeleted and don't return messages that
-	// have already been deleted.
+	// a `Count` method for counting the number of models in a collection for
+	// counting the number of models that satisfy some query and an `Offset` field
```

Comment:
"for counting the number of models in a collection for counting the number of" sounds like a run-on.

Reply:
Oops I really butchered that comment. I blame the line break 😑